code/iaas/api-logic/src/main/java/io/cattle/platform/iaas/api/auth/github/GithubClient.java

```diff
@@ -104,16 +109,44 @@ public GithubAccountInfo getUserAccountInfo(String token) throws IOException {
         return orgInfoList;
     }
 
-    public GithubAccountInfo getUserIdByName(String username) throws IOException {
+    public List<TeamAccountInfo> getOrgTeamInfo(String token, String org) throws IOException {
+        if (StringUtils.isEmpty(token)) {
+            return null;
+        }
+        List<TeamAccountInfo> teamInfoList = new ArrayList<>();
+        List<Map<String, Object>> jsonData = null;
+
+        HttpResponse response = Request.Get(GITHUB_UNAUTH_ORG_URL.get() + org + "/teams").addHeader("Authorization", "token " + token)
+                .addHeader("Accept", "application/json").execute().returnResponse();
+        int statusCode = response.getStatusLine().getStatusCode();
+        if (statusCode != 200) {
+            throw new ClientVisibleException(ResponseCodes.SERVICE_UNAVAILABLE, "GitHubError", "Non-200 Response from Github", "Status code from Github: "
+                    + new Integer(statusCode).toString());
```

Comment:
pedantic: This the same as doing `+ statusCode`  In general `+ statusCode` is better because you do not have the chase of a NullPointerException or NumberFormatException as you do with `new Integer(statusCode).toString()`.  Here you are safe, but in general bad.